CMakeLists.txt

```
@@ -102,6 +105,14 @@ else()
   message("Note: FreeType2 is disabled")
 endif()
 
+if(WITH_UPDATE_CHECK)
+  message("Note: Update check is enabled (default)")
+else()
+  message("Note: Update check is disabled")
+endif()
+
+set(CORSIX_TH_LINK_LUA_MODULES ON)
```

```suggestion
if(VCPKG_TARGET_TRIPLET AND NOT WITH_LUAROCKS)
  set(CORSIX_TH_LINK_LUA_MODULES ON)
else()
  set(CORSIX_TH_LINK_LUA_MODULES OFF)
endif()
```
This works on Mac and Linux. Windows needs a vcpkg sha bump to confirm.